### PR TITLE
feat: add `forceDailySync` to GH code sync

### LINKED ISSUE #6658: Support cron schedule for GitHub workflow

### PR DESCRIPTION
**Pull request by fontanierh**
## Description

fixes https://github.com/dust-tt/dust/issues/6658

Currently, github code sync only works for repos that have pull requests. In some very specific cases, we want to support syncing code daily for repos that do not use pull requests.

- add a `forceDailySync` column to the `github_code_repositories` table
- add a new workflow (`githubCodeSyncDailyCronWorkflow`) to GH connector that just calls a single activity that signals / starts the `githubCodeSyncWorkflow` for a specific repo
- add a CLI that sets `forceDailySync` to true for a specific repo and then launches `githubCodeSyncDailyCronWorkflow` with a daily CRON schedule

To sum up, for a repos we manually decide to daily sync, we set a flag in DB and create a temporal CRON to sync code. , It's a separate workflow that calls an activity which itself uses temporal client to trigger a workflow. But this setup is quite simple, as it is fully additive, doesn't change the existing workflow, and doesn't require cancelling a potentially already running workflow. It essentially simulates a daily pull request webhook.

## Risk

N/A

## Deploy Plan

run migration (connectors) then deploy connectors

### CODE REVIEW

**Review by JulesBelveze — APPROVED** [review]
LGTM, thanks 💯
